css/style.css

```
@@ -762,10 +742,11 @@ a:focus {
 .footer_section {
   background-color: #242424;
   padding: 10px;
-  font-family: "Dosis", sans-serif;
+  font-family: "GlacialIndifferenceRegular", "Arial", "Helvetica", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "GlacialIndifferenceRegular" (font-family-name-quotes)

(font-family-name-quotes)